### PR TITLE
KOA-4691 Add missing content descriptions

### PR DESCRIPTION
**Pull request by marianeum**
This missing content descriptions where missing or mark views as not important for accessibility, where applicable. Also improved the accessibility of the theme switcher a little.

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: app/src/main/res/layout/fragment_flare_pointer_offset.xml [hunk 1]
```diff
@@ -11,10 +11,11 @@
     android:layout_marginBottom="@dimen/bpkSpacingLg"
     app:flarePointerPosition="start">
 
-    <androidx.appcompat.widget.AppCompatImageView
+    <ImageView
```
Comment (marianeum) [inline app/src/main/res/layout/fragment_flare_pointer_offset.xml:14]: android automatically switches to the `AppCompatImageView` - using `ImageView` makes the content description lint check work

File: app/src/main/java/net/skyscanner/backpack/demo/stories/IconsAdapter.kt [hunk 2]
```diff
@@ -51,6 +51,10 @@ class IconsAdapter(
     holder.img.setOnClickListener {
       BpkToast.makeText(holder.itemView.context, names[position], BpkToast.LENGTH_SHORT).show()
     }
+    holder.img.contentDescription = names[position]
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/stories/IconsAdapter.kt:54]: announce it in a nice format - clicking it still announces the full resource name

File: app/src/main/java/net/skyscanner/backpack/demo/SettingsActivity.kt [hunk 3]
```diff
@@ -56,15 +56,16 @@ class SettingsActivity : AppCompatActivity() {
     )
 
     val themeToggle = findViewById<BpkSwitch>(R.id.theme_toggle)
+    val themePicker = findViewById<View>(R.id.theme_picker)
 
-    themeToggle?.apply {
-      setOnCheckedChangeListener { _, isChecked ->
-        onThemeSwitchClicked(isChecked)
-      }
+    themePicker.setOnClickListener {
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/SettingsActivity.kt:61]: set the click listener on the container instead to make the whole row being read out in one go to make it clearer what you're toggling

File: app/src/androidTest/java/net/skyscanner/backpack/map/internal/BpkMapMarkerTest.kt [hunk 4]
```diff
@@ -30,6 +34,13 @@ class BpkMapMarkerTest : BpkSnapshotTest() {
 
   private val bridge = BpkMapMarkersTestBridge.Instance
 
+  @get:Rule
+  var activityRule: ActivityTestRule<AppCompatActivity> =
```
Comment (marianeum) [inline app/src/androidTest/java/net/skyscanner/backpack/map/internal/BpkMapMarkerTest.kt:38]: this is needed to make the appcompat view magic to work

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -46,4 +46,24 @@
   <string name="bottom_nav_trips">Trips</string>
   <string name="bottom_nav_profile">Profile</string>
 
+  <string name="image_rockies_content_description">Image of woman standing on a pier by a lake in the mountains</string>
```
Comment (georgegillams) [inline app/src/main/res/values/strings.xml:49]: As a screen reader will already identify something is an image, there's no need to repeat that information.
Sometimes it's useful to say "photo of..." or "illustration of..." to convey the type of image.

```suggestion
  <string name="image_rockies_content_description">Woman standing on a pier by a lake in the mountains</string>
```

**Review by georgegillams — CHANGES_REQUESTED** [review]
Sorry for taking so long to review. Just a small point from me 👍

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (marianeum) [reply app/src/main/res/values/strings.xml:49]: That is what I thought as well, but when testing it didn't read out that it was an image :(

**Review by georgegillams — APPROVED** [review]

**Pull request merged**